lib/systems/parse.nix

```diff
@@ -361,6 +361,10 @@ rec {
     gnuabin32    = { abi = "n32"; };
     muslabin32   = { abi = "n32"; };
 
+    gnuabielfv2  = { abi = "elfv2"; };
+    muslabielfv2 = { abi = "elfv2"; };
```

Comment:
```suggestion
```

> On the other hand, some packages just don't work with ELFv1, for example the whole musl libc.

In that case we don't need `muslabielfv2`, right?  I mean it's the same thing as `muslabi64` on PowerPC.